alf/algorithms/tasac_algorithm.py

```diff
@@ -147,44 +147,40 @@ def gamma(self):
         """
         return self._gamma.clone()
 
-    def forward(self, experience, value, target_value, train_b):
+    def forward(self, info, value, target_value):
         r"""Calculate the TD loss. The first dimension of all the tensors is the
         time dimension and the second dimesion is the batch dimension.
 
         Args:
-            experience (Experience): experience collected from a replay buffer.
+            info (TasacInfo): TasacInfo collected from train_step().
             value (torch.Tensor): the tensor for the value at each time
                 step. The loss is between this and the calculated return.
             target_value (torch.Tensor): the tensor for the value at each time
                 step. This is used to calculate return.
-            train_b (torch.Tensor): the :math:`\beta` policy actions :math:`\tilde{b}_n`
-                sampled from the current policy. These will be compared to the
-                rollout :math:`\beta` policy actions :math:`b_n`
-                (``experience.rollout_info.action[0]``) to decide the target
-                boostrapping steps.
 
         Returns:
             LossInfo: TD loss with the ``extra`` field same as the loss.
         """
-        if experience.reward.ndim == 3:
+        train_b = info.beta
+        if info.reward.ndim == 3:
             # [T, B, D] or [T, B, 1]
-            discounts = experience.discount.unsqueeze(-1) * self._gamma
+            discounts = info.discount.unsqueeze(-1) * self._gamma
         else:
             # [T, B]
-            discounts = experience.discount * self._gamma
+            discounts = info.discount * self._gamma
 
-        rollout_b = experience.rollout_info.action[0]
+        rollout_b = info.beta
```

Comment:
rollout_b = info.rollout_b

Reply:
hmm. My bad. But strangely, it didn't affect its performance on bipedal walker